tests/RME/RME_test_case.py

```diff
@@ -10,8 +10,7 @@ class RMETestCase(unittest.TestCase):
     """
     Base class for all unit tests using the RME data
     """
-    test_dir = Path(__file__).parent
-    basin_dir = test_dir.joinpath('RME')
+    basin_dir = Path(__file__).parent
```

Comment:
Can you explain moving this file? It seems to go against how SMRF/AWSM are setup to test with basins. Just trying to think of how the format fits with the other tests.

Reply:
I moved this in here because I see this as the next improvement over how SMRF/AWSM does it and would also like to see this change propagate to there. If you look at the class, it basically describes the structure of the basin dir test data and is all you need in order to use the data in tests.

It also clearly separates test data from test logic, moving it in there.

Last argument I see is that the `test_dir` property was removed and this was never used anywhere else, showing that moving the class into the folder is the better place to keep it.

Reply:
Does seem to be cleaner if we move that route. Then in SMRF, the RME/Lakes would be under their presepective directories.

The one major downside I see is discoverability for that class. You pretty much have to know that it's there, as in the SMRF case, it'll be buried a few directories down.

Reply:
I can see this downside addressed with documentation. Currently we have a need for that as a whole, on how they are set up, what parts tests what, or when it is appropriate to updated the gold files.

Reply:
Yes to documentation. But it's way easier to just code.